framework/audio/midi/internal/sequencer.h

```diff
@@ -70,19 +71,19 @@ class Sequencer : public ISequencer
     float playbackSpeed() const override;
     void setPlaybackSpeed(float speed) override;
 
-    void setIsTrackMuted(int t, bool mute) override;
-    void setTrackVolume(int ti, float volume) override;
-    void setTrackBalance(int ti, float balance) override;
+    void setIsTrackMuted(uint16_t t, bool mute) override;
+    void setTrackVolume(uint16_t ti, float volume) override;

```

Comment:
Can we rename these "ti" and "t" to something else?

Reply:
renamed